tests/python/gaia-ui-tests/gaiatest/apps/music/regions/sublist_view.py

```diff
@@ -20,9 +20,8 @@ def switch_to_active_view(self):
         self.marionette.switch_to_frame(self.marionette.find_element(*self._active_view_locator))
 
     def wait_sublist_view_draw(self):
-        active_view = self.marionette.find_element(*self._active_view_locator)
-        Wait(self.marionette).until(expected.element_displayed(active_view))
-        self.marionette.switch_to_frame(active_view)
+        Wait(self.marionette).until(expected.element_displayed(*self._active_view_locator))
+        self.marionette.switch_to_frame(self.marionette.find_element(*self._active_view_locator))
```

Comment:
Can't you use self.switch_to_active_view() instead?
As a matter of fact, perhaps switch_to_active_view() should have this Wait element displayed as well? Then you can just remove this method.
Is this method actually used? I can't seem to find it being used in the source at all.